sfja/src/components/ParentDash/ParentDash.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+// eslint-disable-next-line require-jsdoc
+class ParentDash extends React.Component {
```

Comment:
The real dashboard can be made later; I renamed what used to be here to `ParentHeader`.